src/components/common/EventList.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div v-infinite-scroll="loadMore" infinite-scroll-distance="80">
+    <template v-for="group in groupedEvents">
+      <div :key="group[0].title">
+        <h3>{{ group[0].title }}</h3>
+        <template v-for="item in group">
+          <slot v-bind="item.event" />
+        </template>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import groupBy from "lodash.groupby"
+import orderBy from "lodash.orderby"
+import moment from "moment"
+
+const categories = [
+  {
+    title: "Today",
+    matcher: (event) => {
+      return moment(event.timestamp).isSame(moment(), "day")
```

Comment:
Would be useful to add a small comment how this works (especially `isSame(moment(), "day")`)

Reply:
```suggestion
      // tests if the timestamp has the same day as today
      return moment(event.timestamp).isSame(moment(), "day")
```